docs/rules/no-restricted-class.md

```diff
@@ -0,0 +1,104 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-restricted-class
+description: disallow specific classes
+since: v7.19.0
+---
+# vue/no-restricted-classes
+
+> disallow specific classes
+
+## :book: Rule Details
+
+This rule lets you specify a list of classes that you don't want to allow in your templates.
+
+## :wrench: Options
+
+The simplest way to specify a list of forbidden classes is to pass it directly
+in the rule configuration.
+
+```json
+{
+  "vue/no-restricted-props": ["error", { classes: ["forbidden"] }]
```

Comment:
I think it would be good to be able to accept options in an array like any other `no-restricted-xxx` rule.

```suggestion
  "vue/no-restricted-props": ["error", "forbidden", "forbidden2", "forbidden3"]
```